docs/install.md

```diff
@@ -0,0 +1,70 @@
+# Installing the Knative Pipelines component
+
+Before you can run Knative Pipelines, you must install the Knative Pipelines components
+in your Kubernetes cluster. Use this page to add the component to an existing 
+Kubernetes cluster.
+
+## Before you begin
+
+For complete installation instructions of Knative components, see [Installing Knative](https://github.com/knative/docs/blob/master/install/README.md).
```

Comment:
would be good to mention here that none of the other Knative components are required for Pipelines to work

(there's a common misconception that istio is required for pipelines for example)

Reply:
good point